ppl/cmd/zar/README.md

```diff
@@ -32,26 +32,24 @@ ln -s zq-sample-data/zng
 
 ## ingesting the data
 
-Let's take those logs and ingest them into a directory.   We'll make it
-easier to run all the commands by setting an environment variable pointing
-to the root of the logs tree.
+Let's take those logs and ingest them into a directory.  Often we'd keep our
+archive somewhere like [Amazon S3](https://aws.amazon.com/s3/), but since we're
+just doing a quick test we'll use local temp space.  We'll make it easier to
+run all the commands by setting an environment variable pointing to the root of
+the logs tree.
 ```
-mkdir ./logs
-set ZAR_ROOT=`pwd`/logs
+mkdir /tmp/logs
+ZAR_ROOT=/tmp/logs
```

Comment:
1.  To get this into the environment, `export` is required.
2.  Nit: Might as well use the variable instead of repeating the path.

```suggestion
export ZAR_ROOT=/tmp/logs
mkdir $ZAR_ROOT
```

Reply:
@nwt: Thanks for catching the `export` problem. Everything had been working for me in my runs through the README because I'd had a pre-existing `export ZAR_ROOT` in my `.bashrc` and so it was still having the intended impact even after I was changing the actual value for `ZAR_ROOT`. :roll\_eyes: